rhoshift/main.py

```
@@ -1,36 +1,24 @@
-      O:::::::OOO:::::::O
-      O::::::O   O::::::O
-      O:::::O     O:::::O
-      O:::::O     O:::::O
-      O::::::O   O::::::O
-      O:::::::OOO:::::::O
-       OO:::::::::::::OO 
-         OO:::::::::OO   
-           OOOOOOOOO     
-        """
-        print(banner.center(50))
+    import pyfiglet;
```

Ruff (0.11.9)

Redefinition of unused `pyfiglet` from line 9

Remove definition: `pyfiglet`

(F811)